components/blocks/autofunction.js

```diff
@@ -31,6 +31,7 @@ const Autofunction = ({
   hideHeader,
   deprecated,
   deprecatedText,
+  ancestorFunction,
```

Comment:
I'm not sure if "ancestor" is quite the right term. Makes me think of parent/child hierarchies, maybe related to OOP classes.

Maybe "oldName" would be clearer?

Reply:
Used "oldName" within the component-call in markdown. Used "oldStreamlitFunction" within the JS to match the "function" -> "streamlitFunction" pattern between the component-call and the JS logic.